packages/vite/src/node/plugins/css.ts

```diff
@@ -2780,6 +2780,8 @@ async function compileLightningCSS(
         if (urlReplacer) {
           const replaceUrl = await urlReplacer(dep.url, id)
           css = css.replace(dep.placeholder, () => replaceUrl)
+        } else {
+          css = css.replace(dep.placeholder, () => dep.url)
```

Comment:
Why does the `replace` function use a function as the second parameter? This seems unnecessary. 🤔

Reply:
It's because of https://github.com/vitejs/vite/pull/16306

Reply:
Thanks for the link.
This seems a bit weird, it's the first time I've come across this.

Reply:
Oh, I found the description.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace#specifying_a_string_as_the_replacement